src/components/shared/Navigation.tsx

```diff
@@ -1,16 +1,251 @@
-import { Flex, Text } from "@chakra-ui/react";
+/* eslint-disable react/no-multi-comp */
+import {
+  Flex,
+  Text,
+  Box,
+  Avatar,
+  Button,
+  Stack,
+  Menu,
+  MenuButton,
+  MenuList,
+  MenuItem,
+  Link,
+  Popover,
+  PopoverTrigger,
+  PopoverContent,
+  useColorModeValue,
+  VStack,
+} from "@chakra-ui/react";
+import Image from "next/image";
+import NextLink from "next/link";
+import Logo from "public/images/small_logo.svg";
 
 function Navigation() {
   // Assume that you know this information, nav bar changes based on this information
-  const isLoggedIn = false;
+  const isLoggedIn = true;
   const isChapter = false;
-  const isNonprofit = false;
+  const isNonprofit = true;
+  const npUserName = "liv.2b.girl";
+  const nonProftName = "Liv2BGirl";
+  const chapterName = "Chapter Name";
+
+  //colors
+  const black = "rgba(51, 51, 51, 1)";
+  const slate = "rgba(101, 119, 136, 1)";
+  const blue = "rgba(0, 105, 202, 1)";
 
   return (
-    <Flex height="70px" borderBottom="solid">
-      <Text>Navigation</Text>
-    </Flex>
+    <Box>
+      <Flex
+        bg={useColorModeValue("white", "gray.800")}
+        color={useColorModeValue("gray.600", "white")}
+        maxH={"66px"}
+        py="23px"
+        px="34px"
+        borderBottom={1}
+        borderStyle={"solid"}
+        borderColor={useColorModeValue("gray.200", "gray.900")}
+        align={"center"}
+      >
+        <Flex flex={{ base: 1 }} justify={{ base: "center", md: "start" }}>
```

Comment:
When shrinking browser size to medium breakpoint, the logo will move to the center.  You can fix that by making justify="start".